apps/web/src/app/examples/page.tsx

```diff
@@ -98,7 +98,7 @@ const items = [
     name: "Plaid / Verify Identity",
     author: "zenorocha",
   },
-] satisfies ExampleProps[];
```

Comment:
Why remove the satisfies here? Doesn't this make us lose a bit of intellisense at this point?

Another option is to just set the type of the variable to `ExampleProps[]`